bulbs/poll/migrations/0002_answer.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('poll', '0001_initial'),
```

Comment:
Since these are part of the same initial commit, we should erase these migrations and create `0001_initial.py` again so that both the answer and poll migrations are there